src/web/source.rs

```diff
@@ -729,4 +737,23 @@ mod tests {
             Ok(())
         });
     }
+
+    #[test]
+    fn large_file_test() {
+        wrapper(|env| {
```

Comment:
you can change the config for the test, so you don't have to create a 50 MB file in memory
```suggestion
        wrapper(|env| {
            env.override_config(|config| {
                config.max_file_size = 1;
                config.max_file_size_html = 1;
            });
            // ... 

```

Reply:
Thank you!
Revised the code.